specutils/io/default_loaders/tests/test_jwst_loader.py

```diff
@@ -43,3 +67,45 @@ def test_jwst_loader(tmpdir):
     assert data[0].shape == (100,)
     assert data[1].shape == (120,)
     assert data[2].shape == (110,)
+
+
+def test_jwst_x1d_single_loader(tmpdir, x1d_single):
+    """Test Spectrum1D.read for JWST x1d data"""
+    tmpfile = str(tmpdir.join('jwst.fits'))
+    x1d_single.writeto(tmpfile)
+
+    data = Spectrum1D.read(tmpfile, format='JWST x1d')
+    assert type(data) is Spectrum1D
+    assert data.shape == (100,)
+
+
+def test_jwst_x1d_single_loader_no_format(tmpdir, x1d_single):
+    """Test Spectrum1D.read for JWST x1d data without format arg"""
+    tmpfile = str(tmpdir.join('jwst.fits'))
+    x1d_single.writeto(tmpfile)
+
+    data = Spectrum1D.read(tmpfile)
+    assert type(data) is Spectrum1D
+    assert data.shape == (100,)
+
+
+def test_jwst_x1d_singel_loader_fail_on_multi(tmpdir, x1d_multi):
+    """Make sure Spectrum1D.read on JWST x1d with many spectra errors out"""
+    tmpfile = str(tmpdir.join('jwst.fits'))
+    x1d_multi.writeto(tmpfile)
+
+    with pytest.raises(RuntimeError, match="SpectrumList"):
+        Spectrum1D.read(tmpfile, format='JWST x1d')
```

Comment:
```suggestion
        Spectrum1D.read(tmpfile, format='JWST x1d multi')
```

Should this be failing to load using the `SpectrumList` loader?

Reply:
No, this test to verify that if I use `Spectrum1D.read(file, format="JWST x1d")` on a JWST multi-spec file, then we get a failure telling me to use the `SpectrumList.read()` loader.  And I guess I could remove the format statement here as it is not needed and is probably is causing confusion for the reader.